apps/cricket-score-frontend/README.md

```
@@ -154,23 +156,25 @@
+This frontend connects to the backend inside this monorepo at `apps/cricketscoreapi`. You can:
 - **Use the deployed API** at `https://cricketapi-r9zv.onrender.com` (default)
-- **Run locally** by cloning the backend repo and setting `NEXT_PUBLIC_API_BASE_URL=http://localhost:3001`
-- **Deploy your own** instance using the backend repository's Docker setup
+- **Run locally** by starting the backend and setting `NEXT_PUBLIC_API_BASE_URL=http://localhost:3001`
+- **Deploy your own** instance using the backend's Docker setup
 
 ### Development
 
-1. **Start the development server**
+1. **Start the backend (optional if using hosted API)**
    ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)